cranelift/filetests/filetests/runtests/i128-load-store.clif

```diff
@@ -1,5 +1,7 @@
 test run
 set enable_llvm_abi_extensions=true
+; Disable stack probes since these tests don't require them
+set enable_probestack=false
 target x86_64
```

Comment:
Why do just a few runtests need stack-probing disabled?

Reply:
These tests have stack slots above the threshold where we emit a stack probe. The default strategy is to outline the stack probing to a external function call, but since we never provide an implementation for that anywhere the JIT can't link these tests, so they fail.

I manually disabled stack probing for these which reflects the old behaviour of force disabling it everywhere.